src/index.ts

```diff
@@ -66,7 +66,9 @@ export type DeleteResponse = google.protobuf.Empty;
 export type LogSink = google_config.logging.v2.ILogSink;
 
 export interface CreateSinkRequest {
-  destination: Bucket|Dataset|Topic|string;
+  // destination: Bucket|Dataset|Topic|string;
+  // tslint:disable-next-line no-any
```

Comment:
@JustinBeckwith can you add a comment here indicating why any is being used, and what would be an ideal type definition if it were not for the npm/typescript limitations.